back/controllers/rangeSupervision.js

```diff
@@ -82,6 +82,20 @@ const controller = {
       .send(response.locals.queryResult);
   },
 
+  associationSupervisions: async function getAssociationSupervisions(request, response) {
+    if (response.locals.queryResult.length === 0) {
```

Comment:
To avoid a TypeError, we should check if response.locals.queryResult is undefined, null etc. instead of using .length.

`    if (response.locals.queryResult) {`